app/Http/Controllers/ReviewController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\Review;
+use Illuminate\Http\Request;
+
+class ReviewController extends Controller
+{
+    public function addBookReview (Request $request) 
+    {
+            // Validating the request data
+            $request->validate([
+                'name' => 'required|string|min:1|max:255',
+                'rating' => 'required|integer|min:0|max:5',
+                'review' => 'required|string|max:1000',
+                'book_id' => 'required|integer',
```

Comment:
You should use the exists validation rule on the book\_id, like this:

`exists:books,id` 

That will automatically make sure that the book\_id exists as an id in the books table